src/collections/_documentation/platforms/android/index.md

```diff
@@ -0,0 +1,597 @@
+
+- total storage size
+- free storage size
+- total external storage size
+- free external storage size
+
+Device Info
+
+- Manufacturer
+- Brand
+- Model
+- Model ID
+- Architecture
+
+Battery Level
+
+- Whether the Device is being charged at the moment
+- Whether the phone was online during the error/crash
+
+Screen info
+
+- orientation of the screen
+- screen density
+- hight pixels
+- width pixels
+- resolution
+
+Emulator indicator
+
+- Whether the device is an emulator
+
+## Advanced Usage
+
+### Requirements
+
+For the use of the SDK with the NDK, the minimal required API level is 21. 
+
+If you want to use the SDK without the NDK, you can: 
+
+- disable the NDK as described in the configuration section
+- or use sentry-android-core that doesn't contain the NDK and can be used separately. The minimal required API level for this SDK is 14.
+
+To add the sentry-android-core library, you need to provide the following dependency in your build.gradle file.
+
+```groovy
+// ADD COMPATIBILITY OPTIONS TO BE COMPATIBLE WITH JAVA 1.8
+compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+// ADD SENTRY ANDROID AS A DEPENDENCY
+dependencies {
+        implementation 'io.sentry:sentry-android-core::2.0.+'
+}
+```
+
+If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 21 you can update your manifest as follows:
+
+```xml
+    <!-- Merging strategy for the imported manifests -->
+      <uses-sdk
+            tools:overrideLibrary="io.sentry.android, io.sentry.android.ndk"/>
+```
+
+With these changes, the NDK is going to be used only on the devices with the API level ≥ 21 and the rest of the devices with API level ≥14, but ≤21 will use just the SDK.
+
+### Breadcrumbs
+
+Sentry will automatically record specific events, to provide context to an error.
+
+You can manually add breadcrumbs on other events or disable breadcrumbs.
+
+```java
+Breadcrumb breadcrumb = new Breadcrumb();
+breadcrumb.setMessage("Https Call to Sentry");
+
+Map<String,String> data = new HashMap();
+data.put("url","https:\\sentry.io");
+
+breadcrumb.setData(data);
+
+Sentry.addBreadcrumb(breadcrumb);
+```
+
+For more information, see:
+
+- [Full documentation on Breadcrumbs](https://docs.sentry.io/enriching-error-data/breadcrumbs/)
+- [Debug Issues Faster with Breadcrumbs](https://blog.sentry.io/2016/05/04/breadcrumbs).
+
+### Configuration options
+
+**NDK Integration**
+
+NDK is packed with the SDK and enabled by default with each start of the SDK. 
+
+Alternatively, you can disable the NDK by adding the following line into your manifest.
+
+```xml
+<meta-data android:name="io.sentry.ndk.enable" android:value="false" />
+```
+
+**ANR** 
+
+Whenever the main UI thread of the application is blocked for more than four seconds, the SDK will report the problem to the server.  (Sentry does not report the ANR if the application is in debug mode.)
```

Comment:
```suggestion
Whenever the main UI thread of the application is blocked for more than four seconds, the SDK will report the problem to the server.  (Sentry does not report ANR if the application is in debug mode.)
```

Reply:
I'd also suggest to write 'anr (application no responding)' the first time it shows up

Reply:
Excellent suggestion. 😄